### ISSUE TITLE
libffi8

### ISSUE THREAD

**Issue by CardealRusso** [porteux/porteux#27]
some chromium browsers (specially the ungoogled one) now needs libffi.so.8
fresh install of porteux 0.7 with stable chromium (tested 117) wont launch

**Comment by CardealRusso** [comment]
it seems to be a problem with slackware15 repo nvm, i hope they fix

**Comment by porteux** [comment]
Sorry, I only noticed your issue now.

I tried to fake a symlink from `libffi.so.7` to `libffi.so.8` and it seems the problem is even worse than what we would expect:
```
chromium: /lib64/libm.so.6: version `GLIBC_2.35' not found (required by chromium)
chromium: /usr/lib64/libffi.so.8: version `LIBFFI_BASE_8.0' not found (required by chromium)
chromium: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by chromium)
```

End of the game for Slackware 15.0 :( It's a shame what these developers have been doing by pushing these fast pace updates. Well, the good news is that PorteuX 0.7 current is pretty much rock solid, so maybe you can give it a try. :)

Thanks for the feedback!

**Comment by porteux** [comment]
Oh, the developer version of Chromium works!

Considering that is built by Google while the so-called _stable_ version is built by volunteers in a non-official repository, it's clear that these guys are doing a bad job. I saw your comment on the other repository, and the reaction of the developers was simply ridiculous. Oh, boy... Anyway, I posted something [there](https://github.com/ungoogled-software/ungoogled-chromium-portablelinux/issues). Let's see.
